pages/blog/release-notes-2.2.0.md

```diff
@@ -0,0 +1,64 @@
+---
+title: AsyncAPI Spec 2.2.0 Release Notes
+date: 2021-09-28T06:00:00+01:00
+type: Communication
+tags:
+  - Release Notes
+  - Specification
+cover: /img/posts/release-notes-2.2.0/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+excerpt: '2.2 release of AsyncAPI specification is here. The long-awaited feature for assigning channels to servers is finally here.'
+featured: true
+---
+
+I'm happy to share that AsyncAPI specification 2.2.0 is here. Check out all the goodies that it brings.
+
+> This is a minor release, and it doesn't bring any breaking changes. You can switch to it by modifying the following value in your AsyncAPI file `asyncapi: '2.1.0'` into `asyncapi: '2.2.0'`
+
+## Assigning channels to servers
+
+This feature allows the definition of AsyncAPI documents for applications with more selective use of channels and servers. An example is message protocol adapters that consume messages from one server (say, Kafka) and publish those messages to another server (say, Anypoint MQ).
+
+Now you can add a new **servers** property to [Channel Item Object](https://github.com/asyncapi/spec/blob/2021-09-release/spec/asyncapi.md#channel-item-object). It must be a list of server names provided as a string.
+
+```yaml
+description: This application publishes WebUICommand messages to an AMQP queue on RabbitMQ brokers in the Staging and Production environments.
+servers:
+  - rabbitmqBrokerInProd
+  - rabbitmqBrokerInStaging
+subscribe:
+  message:
+    $ref: "#/components/messages/WebUICommand"
+bindings:
+  amqp:
+    is: queue
+```
+
+Names of servers must match the names of the servers defined in the [Servers Object](https://github.com/asyncapi/spec/blob/2021-09-release/spec/asyncapi.md#serversObject). This new property is optional, so moving from 2.1.0 to 2.2.0 is as easy as changing the specification version in your current AsyncAPI file. Suppose **servers** is absent or empty. In that case, the given channel must be available on all servers defined in the Servers Object, as it was before the 2.2.0 version.
```

Comment:
```suggestion
Names of servers must match the names of the servers defined in the [Servers Object](https://github.com/asyncapi/spec/blob/2021-09-release/spec/asyncapi.md#serversObject). This new property is optional, so moving from 2.1.0 to 2.2.0 is as easy as changing the specification version in your current AsyncAPI file. If **servers** is absent or empty, the given channel must be available on all servers defined in the Servers Object, like the previous version.
```
Somehow the first original shorter sentence was tripping me up... I did this instead as an idea, but maybe also if you made it into a new paragraph like below...

`Let's suppose that **servers** is absent or empty. In that case, the given channel must be available on all servers defined in the Servers Object, like the previous version.`